chrome/content/modules/wzQuicktextVar.jsm

```diff
@@ -153,12 +153,18 @@ wzQuicktextVar.prototype = {
           break;
       }
 
```

Comment:
The old implementation had a special case for image (do nothing)
The new implementation has 3 special cases (additional parameter aType)
If we add the parameter aType for every "get_..."-function there were no special cases anymore.